chat_inPage.js

```diff
@@ -159,7 +160,7 @@ function digestChatData(chatBoxId,chatData) {
 			} else {
 				var contributorLabel = 'label-contributor-'+chatData[key]['contributor']['level'];
 			}
-			var userLabel = '<span class="label label-default '+contributorLabel+'">&nbsp;'+chatData[key]['user']+'&nbsp;</span>&nbsp;&nbsp;';
+			var userLabel = '<span class="label label-default '+contributorLabel+'" onClick="mention(\''+chatBoxId+'\',\''+chatData[key]['user']+'\')">&nbsp;'+chatData[key]['user']+'&nbsp;</span>&nbsp;&nbsp;';
```

Comment:
We both worked on this line, so you'll have to merge in the changes from the master branch and sort out the merge conflict. Let me know if you need help with it.